solidity/contracts/KeepBonding.sol

```diff
@@ -314,6 +327,7 @@ contract KeepBonding {
 
         lockedBonds[bondID] = lockedBonds[bondID].sub(amount);
 
+        /* solium-disable-next-line */
```

Comment:
Isn't it disabled in the global `.soliumrc.json`?

Reply:
Yes, you're right. My VCS underlined it red and that mislead me.